docs/help.md

```diff
@@ -211,5 +211,8 @@ terminal (both `cmd.exe` and powershell), and as such the output could
 be visualized incorrectly if the log lines include utf8 characters. It
 is possible to configure the terminal to visualize those characters
 correctly with the use of [`chcp`](https://ss64.com/nt/chcp.html) by
-executing in the terminal `chcp 65001`. This is a known limitation of
-Node.js.
+executing in the terminal `chcp 65001`. For persistent terminal configuration, the supplied
```

Comment:
Small nit: the line break formatting is out of sync with the rest of the document.